packages/matter-node.js/src/Device.ts

```diff
@@ -43,22 +43,36 @@ import { AdminCommissioningHandler } from "./matter/cluster/server/AdminCommissi
 import { NetworkCommissioningHandler } from "./matter/cluster/server/NetworkCommissioningServer";
 import { AttestationCertificateManager } from "./matter/certificate/AttestationCertificateManager";
 import { CertificationDeclarationManager } from "./matter/certificate/CertificationDeclarationManager";
+import { StorageNodeLocalstorage } from "./persistence/StorageNodeLocalstorage";
+import { PersistenceManager } from "./persistence/PersistenceManager";
 
 const logger = Logger.get("Device");
 
+const storage = new StorageNodeLocalstorage(getParameter("file") || "device.json");
+
 class Device {
     async start() {
         logger.info(`node-matter`);
 
+        const persistenceManager = new PersistenceManager(storage);
+        await persistenceManager.initialize();
+
+        const devicePersistence = persistenceManager.createPersistence("Device");
```

Comment:
I'd call this API: `deviceStorage = storageManager.createContext("device")`

Reply:
adjusted